src/lexer/ReservedWords.ts

```diff
@@ -94,6 +98,8 @@ export const KeyWords: { [key: string]: L } = {
     stop: L.Stop,
     sub: L.Sub,
     to: L.To,
+    try: L.Try,
```

Comment:
Noob question: why are `try` and `endtry` keywords but not reserved words while `throw` is both?